sda/internal/database/db_test.go

```diff
@@ -116,6 +116,10 @@ func (suite *DatabaseTests) SetupTest() {
 		ClientKey:  "",
 	}
 
+	db, err := NewSDAdb(suite.dbConf)
+	assert.Nil(suite.T(), err, "got %v when creating new connection", err)
+	_, err = db.DB.Exec("TRUNCATE sda.files CASCADE")
+	assert.NoError(suite.T(), err)
```

Comment:
why are we adding this now..?

Reply:
This is to prevent false errors when all tests are run together vs when a single test is run.